components/user_settings/display/user_settings_display.jsx

```diff
@@ -110,12 +110,14 @@ export default class UserSettingsDisplay extends React.Component {
 
         this.setState({isSaving: true});
 
-        const preferences = [timePreference, channelDisplayModePreference, messageDisplayPreference, collapseDisplayPreference, linkPreviewDisplayPreference];
-        if (this.state.teammateNameDisplay === this.props.configTeammateNameDisplay) {
-            deletePreferences([teammateNameDisplayPreference]);
-        } else {
-            preferences.push(teammateNameDisplayPreference);
-        }
+        const preferences = [
```

Comment:
Is there a thumb rule about when we change from a single line to multiline assignments and destructuring?

Reply:
Not really. I usually do it when there's 3 or more items, but I don't think I'm consistent since I almost always break `this.setState` into multiple lines even when there's only one field being set.

There's an ESLint rule that we could use if we wanted to make it consistent though